### ISSUE TITLE
NPM tarball contains non-essential files

### ISSUE THREAD

**Issue by jed** [jwagner/smartcrop.js#48]
While [the main smartcrop.js file](https://github.com/jwagner/smartcrop.js/blob/master/smartcrop.js) itself is only 15.7 KB, a local install of `smartcrop` consumes 31MB of disk space, as can be confirmed by running.

```
mkdir test
cd test
npm install smartcrop --prod
du -cksh
```

outputs

```
31M   	total
```

This is due to several inclusions which probably don't need to be shipped to npm, including:

|directory|size|
|---|---|
|/dist|25 MB|
|/release|3.9 MB|
|/.git_|1.9 MB|

Any chance you could pull some of these out?

**Comment by jwagner** [comment]
That's both scary and messy. I switched to white listing files in .npmignore and released, 1.1.1. 44k looks a lot better than 31M. Thanks a lot for letting me know.

**Comment by jed** [comment]
:bow:
